docs/source/index.rst

```diff
@@ -43,13 +44,13 @@ that signifies the GUI toolkit.
 
 The supported values of **ETSConfig.toolkit** are:
 
-* 'qt4' or 'qt': Pyside2 or `PyQt <http://riverbankcomputing.co.uk/pyqt/>`_,
+* 'qt4' or 'qt': PySide2 or `PyQt <http://riverbankcomputing.co.uk/pyqt/>`_,
   which provide Python bindings for the `Qt <http://www.qt.io>`_ framework
   version 4 and 5.
 * 'wx': `wxPython 4 <http://www.wxpython.org>`_, which provides Python bindings
   for the `wxWidgets <http://wxwidgets.org>`_ toolkit.
 * 'null': A do-nothing toolkit, for situations where neither of the other
-  toolkits is installed, but Traits is needed for non-UI purposes.
+  toolkits is installed, but TraitsUI is needed for non-UI purposes.
```

Comment:
I guess the original sentence was targeting dependencies on things that are in the `traits` package.
But it is _probably_ true that traitsui is needed _now_ for non-UI purposes, solely because of the circular dependencies we are still living with. That wasn't the original intent of this sentence though.

Should we remove "but Traits/TraitsUI is needed ..." part of this line?

Reply:
i'm fine with that. people might have different reasons why they want to install TraitsUI and Pyface without a toolkit. I think there's one more place where we mention something of this sort which i will remove as well.